llvm/lib/CodeGen/SelectionDAG/LegalizeVectorTypes.cpp

```diff
@@ -6940,11 +6940,8 @@ SDValue DAGTypeLegalizer::WidenVecOp_MSTORE(SDNode *N, unsigned OpNo) {
     StVal = ModifyToType(StVal, WideVT);
   }
 
-  assert((VT.isScalableVector() ? Mask.getValueType().getVectorMinNumElements()
-                                : Mask.getValueType().getVectorNumElements()) ==
-             (VT.isScalableVector()
-                  ? StVal.getValueType().getVectorMinNumElements()
-                  : StVal.getValueType().getVectorNumElements()) &&
+  assert(Mask.getValueType().getVectorMinNumElements() ==
+          StVal.getValueType().getVectorMinNumElements() &&
```

Comment:
Sorry I made a mistake, it should be:
```suggestion
  assert(Mask.getValueType().getVectorElementCount() ==
          StVal.getValueType().getVectorElementCount() &&
```